uint8 id_subsystem
uint16 id_error
uint16 type_error